libs/containers/Message.lua

```diff
@@ -400,7 +400,7 @@ function get.cleanContent(self)
 			:gsub('<@!?(%d+)>', users)
 			:gsub('<@&(%d+)>', roles)
 			:gsub('<#(%d+)>', channels)
-			:gsub('<a?(:.+:)%d+>', '%1')
+			:gsub('<a?:[%w_]+:(%d+)>', '%1')
```

Comment:
```suggestion
			:gsub('<a?(:[%w_]+:)%d+>', '%1')
```
This needs to be changed to capture the actual name of the emoji instead of the id, this is correct behavior for cleanContent